src/tls/connection.c

```diff
@@ -427,15 +424,13 @@ connection_connect_to_dynamic_wsinstance (Connection *self)
       self->certfile_fd = -1;
     }
 
-  sockaddr_printf (&addr, "%s/https@%s.sock", parameters.wsinstance_sockdir, self->fingerprint.str);
-  debug (CONNECTION, "Connecting dynamic https wsinstance %s:", addr.sun_path);
```

Comment:
This could also get back with `https@%s` and showing the fingerprint?

Reply:
Done (but using the newly-formatted name, since I nixed the printf magic)